homeassistant/components/tuya/select.py

```diff
@@ -27,6 +27,28 @@
             entity_category=EntityCategory.CONFIG,
         ),
     ),
+    # Motion Sensor Light
+    # https://developer.tuya.com/en/docs/iot/gyd?id=Kaof8a8hycfmy
+    "gyd": (
+        SelectEntityDescription(
+            key=DPCode.DEVICE_MODE,
+            name="PIR mode",
+            icon="mdi:arrow-decision-auto",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        SelectEntityDescription(
+            key=DPCode.CDS,
+            name="Light Level",
+            icon="mdi:theme-light-dark",
+            entity_category=EntityCategory.CONFIG,
+        ),
+        SelectEntityDescription(
```

Comment:
Please provide translation functionality for these values, by defining a custom device class and add the translations to the strings file.

Reply:
Translation facility added